modules/swagger-codegen/src/main/resources/python/tornado/rest.mustache

```diff
@@ -41,30 +39,10 @@ class RESTClientObject(object):
 
     def __init__(self, configuration, pools_size=4, maxsize=4):
         # maxsize is number of requests to host that are allowed in parallel
-        # ca_certs vs cert_file vs key_file
-        # http://stackoverflow.com/a/23957365/2985775
 
-        # ca_certs
-        if configuration.ssl_ca_cert:
-            ca_certs = configuration.ssl_ca_cert
-        else:
-            # if not set certificate file, use Mozilla's root certificates.
-            ca_certs = certifi.where()
-
-        if hasattr(ssl, 'create_default_context'):
-            # require Python 2.7.9+, 3.4+
-            self.ssl_context = ssl.create_default_context()
```

Comment:
Does the actual tornado client handle creating the ssl_context now, thereby making this redundant?

Reply:
Yes, Tornado create context: https://github.com/tornadoweb/tornado/blob/v4.5.2/tornado/simple_httpclient.py#L248